### DISCUSSION TITLE
Certification Tax Residence GitHub, Inc.

### DISCUSSION THREAD

**Discussion by AdamDrozapol** [community/community#131483] [Enterprise]
Hi,
my accounting manager's needs a Certification Tax Residence GitHub, Inc.
My organization is from Poland.
Where can I find it or get it ?

**Comment by ghostinhershell** [comment]
Hi @AdamDrozapol! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by ramaminarti14-collab** [comment]
[](url) https://github.com/orgs/community/discussions/131483?sort=new#discussion-6914132

**Comment by ramaminarti14-collab** [comment]
[](url)https://github.com/orgs/community/discussions/131483?sort=new#discussion-6914132
